Example/Emission/AppDelegate.m

```diff
@@ -63,45 +63,50 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 
   emission.switchBoardModule.presentNavigationViewController = ^(UIViewController * _Nonnull fromViewController,
```

Comment:
That sounds like a nice change for @sarahscott to do post v1 to dip her toes into RN native modules.

https://github.com/artsy/emission/issues/70